packages/model/Dataset.ts

```diff
@@ -43,6 +44,9 @@ export function DatasetMixin<Base extends Constructor>(Resource: Base): Mixin {
 
     @property.literal({ path: schema.datePublished, type: Date, datatype: xsd.date })
     published!: Date
+
+    @property.literal({ path: cc.isHiddenCube, datatype: xsd.boolean })
+    isHiddenCube!: boolean
```

Comment:
As discussed, I would move and duplicate this field in order to memoize the current cube setting for the event of rerunning a job

1. Add to [`Project`](https://github.com/zazuko/cube-creator/blob/dd4cba3934efbb20e2d4e91a909b9ab3845dfd21/packages/model/Project.ts#L18)
2. Add to [`PublishJob`](https://github.com/zazuko/cube-creator/blob/dd4cba3934efbb20e2d4e91a909b9ab3845dfd21/packages/model/Job.ts#L35)

In order to connect those pieces you'd need to modify the project shape (create and update) and then when then copy to the job when it is created. Look for `createPublishJob` which the domain logic entrypoint.

Finally, you'd need to add or modify some step in `publish.ttl`. There ar already some steps which generate additional metadata so it should not be a big deal.

Reply:
I moved the flag from Cube Metadata to Project.

Now I have the issue, that it fails on `<Create Project>`. The validation reports seems to be empty - what does that mean?

![image](https://github.com/zazuko/cube-creator/assets/877232/c4860ce2-ad3e-4a60-b3b1-679dc14fdaa8)